src/pyload/plugins/downloaders/SKnigavuhe.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+
+from ..base.simple_downloader import SimpleDownloader
+
+
+class SKnigavuhe(SimpleDownloader):
+    __name__ = "SKnigavuhe"
+    __type__ = "downloader"
+    __version__ = "0.1"
+    __status__ = "testing"
+    __pattern__ = (
```

Comment:
`__pattern__ ` must not be surrounded by round brackets

Reply:
It does automatically by black due to line length > 88(89 to be exact). I guess linter will fail if i change that.